docker-compose/technical/init-lines-catalog.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+function init_lines_catalog()
+{
+  CATALOG_EXAMPLE=/init-data/lines-catalog.json
+  if [ -f $CATALOG_EXAMPLE ]; then
+      curl -X 'POST' -f -s -o /dev/null 'http://network-modification-server/v1/network-modifications/catalog/line_types'  -H "Content-Type: application/json" -H 'accept: application/json' --data @${CATALOG_EXAMPLE}
```

Comment:
* No need of "accept" header because [endpoint return `ResponseEntity<Void>`](//github.com/gridsuite/network-modification-server/blob/main/src/main/java/org/gridsuite/modification/server/NetworkModificationController.java#L193-L196).
* escape the variable in case the path contain spaces

```suggestion
      curl -X 'POST' -f -s -o /dev/null 'http://network-modification-server/v1/network-modifications/catalog/line_types'  -H "Content-Type: application/json" --data "@$CATALOG_EXAMPLE"
```